### PR TITLE
Update Issue Templates

### PR DESCRIPTION
**Pull request by maximousblk**
Updates:

two new issues added
- Documentation issue
- Question/Help

new labels needed:
- `type: docs`
- `type: help`

cc @zorbyte

### CODE REVIEW

**Comment by zorbyte** [comment]
We have meta: documentation

**Comment by maximousblk** [comment]
an issue template as for a type of issues so makes more to have `type:` scope

**Comment by zorbyte** [comment]
I understand where you are coming from, but having two tags with lots of overlap is just more work. If we use the meta tag, we all know what is happening.

**Comment by zorbyte** [comment]
Also, the question template is not required, all questions should go to our discussions forum instead.

File: .github/ISSUE_TEMPLATE/documentation_issue.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: 📚 Documentation Issue
+about: Report an issue related to documentation
+labels: 'type: docs, status: needs triage'
```
Comment (zorbyte) [inline .github/ISSUE_TEMPLATE/documentation_issue.md:4]:
```suggestion
labels: 'meta: documentation, status: needs triage'
```

Use meta instead, `type:` has too much overlap with it.

**Comment by zorbyte** [comment]
I should add that I understand that its just a way to get people to move to discussions, but we should discourage it altogether.

**Comment by maximousblk** [comment]
updated as per request

**Pull request merged**